50 migrate to new sdk (#51)
cloudant-labs/kafka-connect-cloudant: CHANGES.md

# UNRELEASED

> - [NEW] The underlying Cloudant client library is now EOL. Refactored implementation and test suite to

Reviewer: Nit, it is new, but it's also an upgrade and I think we can keep it short and sweet.
```suggestion
- [UPGRADED] Refactored to use the new `cloudant-java-sdk` library.
```
